app-misc/utimer/utimer-0.4-r2.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="A command line timer and stopwatch"
+HOMEPAGE="https://sourceforge.net/projects/utimer/"
+SRC_URI="http://utimer.codealpha.net/dl.php?file=${P}.tar.gz -> ${P}.tar.gz"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug nls"
+RESTRICT="test" #630952
```

Comment:
Restricting the tests doesn't really fix the bug, only hides it.  Could you add a comment stating why does the test fail, or if there is anything wrong with the test itself?